# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/html/semantics/embedded-content/the-img-element/image-loading-lazy-data-url-to-https.html.ini
# Commit: 425cbe3c033b
# Full Hash: 425cbe3c033bfc25795979513ebd2bb9fda7338d
# Author: moz-wptsync-bot <wptsync@mozilla.com>
# Date: 2024-11-28 21:54:11
# Regressor Bug: 1933829
# File Overlap Count: 1
# Description:
#   Bug 1933829 - [wpt-sync] Update web-platform-tests to 2d5d222739fdf91743f8cbc58094976a613194e7, a=testonly
#   
#   MANUAL PUSH: wpt sync bot
#   
#   wpt-head: 2d5d222739fdf91743f8cbc58094976a613194e7
# ==============================================================================

diff -r 732bcd7ac971 -r 425cbe3c033b testing/web-platform/meta/html/semantics/embedded-content/the-img-element/image-loading-lazy-data-url-to-https.html.ini
--- a/testing/web-platform/meta/html/semantics/embedded-content/the-img-element/image-loading-lazy-data-url-to-https.html.ini	Wed Nov 27 22:05:28 2024 +0000
+++ b/testing/web-platform/meta/html/semantics/embedded-content/the-img-element/image-loading-lazy-data-url-to-https.html.ini	Wed Nov 27 22:05:56 2024 +0000
@@ -1,6 +1,4 @@
 [image-loading-lazy-data-url-to-https.html]
   expected:
     if debug and (os == "win") and (processor == "x86_64") and not swgl: [PASS, FAIL]
-    if debug and (os == "linux") and not fission and swgl: [PASS, FAIL]
-    if debug and (os == "linux") and not fission and not swgl: [PASS, FAIL]
-    if debug and (os == "linux") and fission: [PASS, FAIL]
+    if debug and (os == "linux"): [PASS, FAIL]